openid-4-verifiable-presentations-1_0.md

```diff
@@ -2237,6 +2251,16 @@ The following is a non-normative example of the payload of a JWT-based W3C Verif
 
 <{{examples/credentials/jwt_vc.json}}
 
+#### Metadata
+
+The `vp_formats` parameter of the Verifier metadata or the `vp_formats_supported` parameter of the Wallet metadata MUST have the key `jwt_vp_json` or `jwt_vc_json`, and the value MUST be an object consisting of the following name/value pair:
```

Comment:
should we just reference the Credential Format Identifier here? (and in the other sections)